bool torque_control_mode_enable
---

